Sources/StreamChat/WebSocketClient/Events/ChannelEvents.swift

```diff
@@ -4,60 +4,210 @@
 
 import Foundation
 
+/// Triggered when a channel is updated.
 public struct ChannelUpdatedEvent: ChannelSpecificEvent {
-    public let cid: ChannelId
+    /// The identifier of updated channel.
+    public var cid: ChannelId { channel.cid }
+    
+    /// The updated channel.
+    public let channel: ChatChannel
+    
+    /// The user who updated the channel.
+    public let user: ChatUser?
```

Comment:
I think in some events it makes sense to name `updatedBy`  and `deletedBY` like you mentioned before, it provides better context 🤔